OKR/PROJECT.md

```diff
@@ -2,6 +2,26 @@
 
 We frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
+## 2019 Q4
+- **O: IPFS Project prepared for 2020 growth** 
+  - Onboard 4+ new contributors with skills around Testing infra, Gateways support, go-ipfs swe, and docs content strategy
+  - 2020 planning identifies our next area of focus and starts building toward it
```

Comment:
Wondering if could be at some point "<Visionary Goal part Deux>"

Reply:
Good idea!
```suggestion
  - 2020 planning identifies our next area of focus and updates our roadmap by Nov 16
```